ios/Classes/TiBarcodeModule.mm

```diff
@@ -309,7 +309,10 @@ -(void)capture:(id)args
 			[controller setSoundToPlay:soundURL];
 		}
 	}
-	
+    if(([UIDevice currentDevice].orientation == UIDeviceOrientationLandscapeLeft) || ([UIDevice currentDevice].orientation == UIDeviceOrientationLandscapeRight)||([UIDevice currentDevice].orientation == UIDeviceOrientationFaceUp) || ([UIDevice currentDevice].orientation == UIDeviceOrientationFaceDown)|| ([UIDevice currentDevice].orientation == UIDeviceOrientationUnknown)|| ([UIDevice currentDevice].orientation == UIDeviceOrientationPortraitUpsideDown))
+    {
+        [[UIDevice currentDevice] setValue:[NSNumber numberWithInteger: UIInterfaceOrientationPortrait] forKey:@"orientation"];
+    }
```

Comment:
Agreed. Actually in OverlayView landscape mode  is not handled. Now I am trying to give support for that as well , so that rotation can also be possible.